### ISSUE TITLE
Changing language in calibration process

### ISSUE THREAD

**Issue by guidobender** [MaslowCNC/GroundControl#785]
Hi,
I ran into some confusion when calibrating with the 1.26 ground control and triangular sled. I would like to suggest changing some of the wording to increase clarity.
Please let me know if you have questions.
Thanks
Tschöhö
Superbender

**Current language of step to adjust chain length after motor-to-motor distance measurement:**

Now we are going to adjust the chains to a known length.

If your chains are not attached place the first link of the chain on the vertical sprocket tooth.
If your chains are already in place they may retract to the target length.

The correct length of first the left and then the right chain will be measured out.
…(more text)

**Text that I would suggest:**

If your chains are already in place they may retract to the target length.

If your left chain is still attached to the right motor from the length measurement motor-to-motor, remove it from the right motor without changing the position it has on the left motor.

For chains that are not attached to a motor (typical calibration = right chain) place the first link of the chain on the vertical sprocket tooth.

…(more text)

**Comment by BarbourSmith** [comment]
That is an excellent idea. Thank you for giving me exact wording for what to change I will create a pull request to propose the change

**Comment by BarbourSmith** [comment]
I created #786 to add the change. Let me know what you think, and give it a 👍 once the robot comments if it looks good

**Comment by GeroBH** [comment]
To avoid the consitant confusion with bottom feed, replace the top feed
bracket picture with the bottom feed triangular pic?

**Comment by BarbourSmith** [comment]
@GeroBH I agree. Having an updated picture is a good idea. Unfortunately I am not in a position to take one right away, but I will keep it on my todo list. Does anyone else have a good picture with the sled centered that we could use there?

**Comment by guidobender** [comment]
Below are a couple of images from my machine. I hope that helps.
I have another suggestion regarding the calibration. Will open another issue shortly.
Very cool to collaborate in this way. You guys rock.

![img_4026](https://user-images.githubusercontent.com/5915936/51511965-b8087880-1dc0-11e9-9656-10189656acae.jpg)
![img_4025](https://user-images.githubusercontent.com/5915936/51511966-b8087880-1dc0-11e9-963a-59d8bf249a33.jpg)

**Comment by GeroBH** [comment]
merged commit [`c79e67d`](/MaslowCNC/GroundControl/commit/c79e67de9afe508f8d1d27218cccaa303c8a055c) into master [13 days ago](/MaslowCNC/GroundControl/pull/788#event-2110933936)
So can be closed?

**Comment by BarbourSmith** [comment]
I vote to close
